tests/test_transfer.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import pytest
+import time
+
+from tests.testing import azure
+from adlfs.transfer import ADLTransferClient
+
+
+@pytest.mark.skipif(True, reason="skip until resolve timing issue")
```

Comment:
This can be address in a future PR. This particular test is a similar version once contained in the `multithread` module.